README.md

```diff
@@ -1,18 +1,680 @@
+
+- Correct: "Install Temporal"
+- Incorrect: "Installing Temporal"
+
+#### Sentence casing in headings
+
+Use sentence casing for titles and headings.
+Sentence casing means that only the first letter of the first word and proper nouns are capitalized.
+
+- Correct: "How to get started with Temporal"
+- Incorrect: "How To Get Started With Temporal"
+
+### What is the philosophy around versioning the documentation?
+
+The Temporal Platform includes many different components and core dependencies. Each of these components is often independently versioned and documents their stability and support for their own dependencies. For example, the [Temporal Go SDK reference](https://pkg.go.dev/go.temporal.io/sdk?tab=versions) provides that view.
+
+The goal of this information set, in regards to versioning, is to remain “current”. That is, this information should serve the needs of the Platform’s user base as best it can based on what is/has recently happened across all the components of the Temporal Platform.
+We should explicitly call out support, stability, and dependency information whenever possible inline with the rest of the information. This contrasts with publishing versions of documentation sets to match the versioned components of the platform.
+
+We use the `ssdi` metadata tag in `docs-src` nodes to explicitly call out support, stability, and dependency information when needed.
+
+#### How to explicitly identify support, stability, and dependency info
+
+Use the `ssdi` metadata tag.
+
+Add this metadata tag to the information node’s metadata tag in the `/docs-src` directory.
+
+Example:
+
+```markdown
+---
+id: what-is-x
+// ...
+ssdi:
+  - Introduced in [Temporal Server version x.xx.x](/path/to/release/notes/)
+  - Available in [Temporal CLI version x.xx.x](/path/to/release/notes/)
+  - Released in Temporal Cloud on <date>
+---
+```
+
+### How to auto format files
+
+Run `yarn format` .
+
+If you do not run this command locally and put up a Pull Request, a Github action runs the formatting.
+
+### How to run the Assembly Workflow
+
+First, you must have a Namespace setup to operate with. We recommend using the [Temporal CLI’s development Server](https://docs.temporal.io/cli/#starting-the-temporal-server). In a separate terminal run:
+
+```bash
+temporal server start-dev
+```
+
+Then again in a separate terminal, run the Worker in `/assembly`
+
+```bash
+cd assembly
+./worker.js
+```
+
+In another terminal start the Workflow from the root of the repo:
+
+```bash
+yarn assemble
+```
+
+After the Workflow completes, we recommend running the auto formatter:
+
+```bash
+yarn format
+```
+
+**TIP**: Make your changes in the **source files** that are named in the configuration file. For example, if you found a typo in the Concept section *What is a Task?*, open [docs/concepts/what-is-a-task.md](https://github.com/temporalio/documentation/blob/main/docs/concepts/what-is-a-task.md) and make the edit directly there.
```

Comment:
```suggestion
**Make your changes in the _source files_  named in the configuration file.**
For instance, if you find a typo under "What is a Task?" (located in the Workers section of the Temporal docs website, locate the source node in `/docs-src` and make any corrections there.
```

Reply:
Removing the link to the example is just far less clear imo.